lib/taskana-core-test/src/test/java/acceptance/task/create/CreateTaskAccTest.java

```diff
@@ -0,0 +1,950 @@
+    assertThat(createdTask.getPriority()).isEqualTo(0);
+    assertThat(createdTask.getClassificationSummary()).isEqualTo(defaultClassificationSummary);
+    assertThat(createdTask.getClassificationSummary().getDomain()).isEqualTo("DOMAIN_A");
+  }
+
+  @WithAccessId(user = "user-1-2")
+  @Test
+  void should_ThrowException_When_WorkbasketDoesNotExist() {
+    Task task = taskService.newTask("UNKNOWN");
+    task.setClassificationKey(defaultClassificationSummary.getKey());
+    task.setPrimaryObjRef(defaultObjectReference);
+    task.setOwner("user-1-2");
+
+    ThrowingCallable call = () -> taskService.createTask(task);
+    WorkbasketNotFoundException e = catchThrowableOfType(call, WorkbasketNotFoundException.class);
+    assertThat(e.getId()).isEqualTo("UNKNOWN");
+  }
+
+  @WithAccessId(user = "user-1-1")
+  @Test
+  void should_TestGetException_When_AppendIsNotPermitted() throws Exception {
+    WorkbasketAccessItemBuilder.newWorkbasketAccessItem()
+        .workbasketId(defaultWorkbasketSummary.getId())
+        .accessId("user-1-1")
+        .permission(WorkbasketPermission.OPEN)
+        .permission(WorkbasketPermission.READ)
+        .buildAndStore(workbasketService, "admin");
+    Task task = createDefaultTask();
+
+    ThrowingCallable call = () -> taskService.createTask(task);
+
+    NotAuthorizedOnWorkbasketException e =
+        catchThrowableOfType(call, NotAuthorizedOnWorkbasketException.class);
+    assertThat(e.getCurrentUserId()).isEqualTo("user-1-1");
+    assertThat(e.getWorkbasketId()).isEqualTo(defaultWorkbasketSummary.getId());
+    assertThat(e.getRequiredPermissions()).contains(WorkbasketPermission.APPEND);
+  }
+
+  @WithAccessId(user = "user-1-2")
+  @TestFactory
+  Stream<DynamicTest>
+      should_ThrowException_When_MandatoryPrimaryObjectReferenceIsNotSetOrIncomplete() {
+    List<Pair<String, ObjectReference>> list =
+        List.of(
+            Pair.of("Object Reference is Null", null),
+            Pair.of(
+                "Value of Object Reference is Null",
+                createObjectReference("COMPANY_A", "SYSTEM_A", "INSTANCE_A", "VNR", null)),
+            Pair.of(
+                "Type of Object Reference is Null",
+                createObjectReference("COMPANY_A", "SYSTEM_A", "INSTANCE_A", null, "1234567")),
+            Pair.of(
+                "Company of Object Reference is Null",
+                createObjectReference(null, "SYSTEM_A", "INSTANCE_A", "VNR", "1234567")));
+    ThrowingConsumer<Pair<String, ObjectReference>> testCreateTask =
+        t -> {
+          Task task = taskService.newTask(defaultWorkbasketSummary.getId());
+          task.setClassificationKey(defaultClassificationSummary.getKey());
+          task.setOwner("user-1-2");
+          if (t.getRight() != null) {
+            task.setPrimaryObjRef(t.getRight());
+          }
+
+          ThrowingCallable call = () -> taskService.createTask(task);
+
+          assertThatThrownBy(call)
+              .describedAs(
```

Comment:
Can you please delete the describedAs everywhere in this class? The content of the test should be clear from the test name and test code